spring-xd-module/src/main/java/org/springframework/xd/module/spark/streaming/SparkStreamingDriverModule.java

```diff
@@ -56,13 +59,14 @@
 import org.springframework.xd.module.ModuleType;
 import org.springframework.xd.module.core.ResourceConfiguredModule;
 import org.springframework.xd.module.options.ModuleOptions;
-import org.springframework.xd.spark.streaming.Processor;
+import org.springframework.xd.spark.streaming.java.Processor;
 import org.springframework.xd.spark.streaming.SparkConfig;
 import org.springframework.xd.spark.streaming.SparkMessageSender;
-import org.springframework.xd.spark.streaming.SparkStreamingModuleExecutor;
+import org.springframework.xd.spark.streaming.java.SparkStreamingModuleExecutor;
+import org.springframework.xd.spark.streaming.SparkStreamingSupport;
 
 /**
- * The driver that adapts an implementation of {@link org.springframework.xd.spark.streaming.Processor} to be executed as an XD module.
+ * The driver that adapts an implementation of {@link org.springframework.xd.spark.streaming.java.Processor} to be executed as an XD module.
```

Comment:
should mention that this adapts either the java or scala Processor implementation (not just java)